app/src/androidTest/java/nextstep/signup/fixture/FirstColumnTest.kt

```diff
@@ -0,0 +1,26 @@
+package nextstep.signup.fixture
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+fun FirstColumnTest(text: String) {
+    Column(
+        modifier = Modifier.testTag("이름")
+    ) {
+        Text(text = "$text", color = Color.Gray)
```

Comment:
```suggestion
        Text(text = text, color = Color.Gray)
```
사소하지만 text 파라미터 타입 자체가 String이라 위처럼 바꿀 수 있겠네요~!

Reply:
헉 testFixture들이 전부 저렇게 되어있었네요..? 제가 왜그랬을까요 🤣
지금은 고쳤습니다~